app/components/newsletter/newsletter-header-overview.js

```diff
@@ -60,36 +60,42 @@ export default class NewsletterHeaderOverviewComponent extends Component {
 
   @task
   *publishToMail() {
-    yield this.ensureMailCampaign();
-
-    if (this.mailCampaign?.isSent) {
-      this.toaster.error(this.intl.t('error-already-sent-newsletter'));
-    } else {
-      if (yield this.validateMailCampaign()) {
-        try {
-          yield this.newsletterService.sendMailCampaign(this.mailCampaign.campaignId);
-          this.mailCampaign.sentAt = new Date();
-          yield this.mailCampaign.save();
-          yield this.args.meeting.belongsTo('mailCampaign').reload(); // TODO Why?
-          this.toaster.success(this.intl.t('success-publish-newsletter-to-mail'));
-        } catch(e) {
-          this.toaster.error(
-            this.intl.t('error-send-newsletter'),
-            this.intl.t('warning-title')
-          );
+    try {
+      yield this.ensureMailCampaign();
+
+      if (this.mailCampaign?.isSent) {
+        this.toaster.error(this.intl.t('error-already-sent-newsletter'));
+      } else {
+        if (yield this.validateMailCampaign()) {
+          try {
+            yield this.newsletterService.sendMailCampaign(this.mailCampaign.campaignId);
+            this.mailCampaign.sentAt = new Date();
+            yield this.mailCampaign.save();
+            yield this.args.meeting.belongsTo('mailCampaign').reload(); // TODO Why?
```

Comment:
Regarding this TODO.
This was probably a fix in some cases, hard to tell what the exact reason/cause was.
I vaguely remember issues where `meeting` did not get updated locally with the `mail-campaign` and in some cases a `meeting.save()` happened later, deleting the relation between mail-campaign and meeting.